src/Layers/xrRender/blenders/dx11RainBlender.cpp

```diff
@@ -5,6 +5,9 @@ void CBlender_rain::Compile(CBlender_Compile& C)
 {
     IBlender::Compile(C);
 
+    auto const PreviousSmapSize = RImplementation.o.smapsize;
+    RImplementation.o.smapsize = RImplementation.o.rain_smapsize;
```

Comment:
Please, don't put it into `o`, put it into RImplementation, like this:
```suggestion
    RImplementation.m_CurrentSMAPSize = RImplementation.o.rain_smapsize;
```
Behaviour almost exactly like `RImplementation.m_MSAASample`.